src/main/java/net/snowflake/client/core/SFLoginInput.java

```diff
@@ -249,8 +249,7 @@ public int getSocketTimeoutInMillis() {
     return (int) socketTimeout.toMillis();
   }
 
-  @SnowflakeJdbcInternalApi
-  public SFLoginInput setSocketTimeout(Duration socketTimeout) {
+  SFLoginInput setSocketTimeout(Duration socketTimeout) {
```

Comment:
Is it fine to reduce the visibility here?

Reply:
we can reduce on internal methods